tests/test_s3_specific.py

```diff
@@ -29,7 +31,9 @@ def test_transfer_config(s3_rig, tmp_path):
 
     # download
     client.set_as_default_client()
-    p = s3_rig.create_cloud_path("dir_0/file0_0.txt")
+    p = s3_rig.create_cloud_path(f"{s3_rig.test_dir}/dir_0/file0_0.txt")
+    p.write_text("test file")
```

Comment:
If you keep the original `s3_rig.create_cloud_path("dir_0/file0_0.txt")` then `dir_0/file0_0.txt` should already exist because it's copied over when the rig is set up.